src/main/scala/sigmastate/Values.scala

```diff
@@ -424,7 +426,12 @@ object Values {
 
   case class ConcreteCollection[V <: SType](items: IndexedSeq[Value[V]], elementType: V)
     extends EvaluatedCollection[V] {
-    override val opCode: OpCode = ConcreteCollectionCode
+    override val opCode: OpCode =
```

Comment:
@ergomorphic I don't like having separate op code for ConcreteCollection with boolean constants, but I could not found a better way. How does it look to you?

Reply:
@greenhat We need to decide about serialization format for the following cases:
- `Constant[SCollection[SBoolean.type]`
- `ConcreteCollection(True, False, True,...)` when all values are constants
- `ConcreteCollection(True, SomeValue, False, SomeOtherValue,...)`

Reply:
@ergomorphic 
* `Constant[SCollection[SBoolean.type]` is bit encoded (see my comment https://github.com/ScorexFoundation/sigmastate-interpreter/pull/178/files#r196445633 )
* `ConcreteCollection(True, False, True,...)` is bit encoded and handled via separate op code here (just below this exact comment) with `ConcreteCollectionBooleanConstantSerializer` 
* as for `ConcreteCollection(True, SomeValue, False, SomeOtherValue,...)` I suggest leave it as is (each value is encoded separately).

Reply:
@greenhat agree